src/utils.mjs

```diff
@@ -167,3 +168,20 @@ export function shouldProcessFile(npath) {
   // see https://github.com/withastro/astro/blob/0fec72b35cccf80b66a85664877ca9dcc94114aa/packages/astro/src/content/utils.ts#L253
   return !npath.split(path.sep).some((p) => p && p.startsWith("_"));
 }
+
+/** @type {Record<string, import('./utils.d.ts').MatterData>} */
+const matterCache = {};
+const matterCacheEnabled = process.env.ARRML_MATTER_CACHE_DISABLE !== "true";
```

Comment:
Already merged as I don't think it's a huge problem, but we should probably think about something to remove this so it can't accidentally be used when installed...